src/transformers/models/pop2piano/feature_extraction_pop2piano.py

```diff
@@ -0,0 +1,288 @@
+# coding=utf-8
+# Copyright 2023 The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Feature extractor class for Pop2Piano"""
+
+from typing import List, Optional, Union
+
+import librosa
+import numpy as np
+import scipy
+import torch
+from torch.nn.utils.rnn import pad_sequence
+
+from ...audio_utils import fram_wave, get_mel_filter_banks, stft
```

Comment:
The audio_utils have been updated and the functions you're using here are now deprecated. Please switch to the `mel_filterbank()` and `spectrogram()` functions. In case they don't do 100% what you need, let me know and I'll add the missing functionality (but I expect them to work correctly).